docs/content/integrations/airflow.mdx

```diff
@@ -1,24 +1,25 @@
 ---
-title: Dagster with Airflow | Dagster
+title: "Dagster + Airflow | Dagster Docs"
 description: The dagster-airflow package allows you to import Airflow DAGs into Dagster jobs and assets, as well as trigger job runs from Airflow.
 ---
 
 # Using Dagster with Airflow
 
 <CodeReferenceLink filePath="examples/with_airflow" />
 
-The [`dagster-airflow`](/\_apidocs/libraries/dagster-airflow) library provides interoperability between Dagster and Airflow. The main scenarios for using the Dagster Airflow integration are:
+The [`dagster-airflow`](/\_apidocs/libraries/dagster-airflow) library provides interoperability between Dagster and Airflow. Using `dagster-airflow`, you can:
 
-- You want to do a lift-and-shift migration of all your existing Airflow DAGs into Dagster jobs/Software-defined Assets (SDAs)
-- You want to trigger Dagster job runs from Airflow
+- Incorporate an Airflow Directed Acyclic Graph (DAG) directory as a Dagster repository, allowing you to do a lift-and-shift migration of your existing Airflow DAGs into Dagster jobs or Software-defined Assets (SDAs)
```

Comment:
Thoughts on linking to the relevant sections of the reference for each of these?

Also, I think the distinction between the first bullet and the third bullet is pretty subtle. They're doing basically the same thing, except the first bullet is doing it for all DAGs, while the third bullet is doing it just for individual DAGs.  I think the wording here could make this distinction more clear.